environments/prod/prod.tfvars

```diff
@@ -1845,6 +1845,11 @@ frontends = [
         operator       = "Equals"
         selector       = "connect.sid"
       },
+      {
+        match_variable = "RequestCookieNames"
+        operator       = "Equals"
+        selector       = "ccpay-bubble-cookie-preferences"
```

Comment:
It's probably not for this PR, but usually cookie names should be as small as possible.  Recommend renaming it in future.

Reply:
sure @adusumillipraveen . We will ensure it going forward.